azure/master-to-test.pipeline.yml

```diff
@@ -2,7 +2,7 @@
 # https://aka.ms/yaml
 
 trigger:
-- master
+- feature/group-update
```

Comment:
Hmm.. Do we want to revert this back to master after review is done? Anyways I think it was a good way to get it to higher environments as a demo.

Reply:
Yes, we do 👍